Bright Tides/Assets/Scripts/Scriptables/EntityAttributes.cs

```diff
@@ -10,9 +10,6 @@ public class EntityAttributes : ScriptableObject {
 		
```

Comment:
This is related to a different issue.  I would like to these stats for now because might use them for applying upgrades.

Reply:
okay I'll revert that one